cmd/ddev/cmd/exec_test.go

```diff
@@ -71,8 +71,12 @@ func TestCmdExec(t *testing.T) {
 	// Test with raw cmd
 	out, err = exec.RunHostCommand(DdevBin, "exec", "--raw", "--", "ls", "/usr/local")
 	assert.NoError(err)
+	assert.True(strings.HasPrefix(out, "bin\netc\ngames\ninclude"), "expected '%v' to start with 'bin\\netc\\games\\include'", out)
```

Comment:
nit: These two assertion messages don't show correctly, and the expectation and actual are switched. this can be written much more easily. (Just the failure message is bad.)

```suggestion
	assert.True(strings.HasPrefix(out, "bin\netc\ngames\ninclude"), `expected '%v' to start with '%s'`, out, `bin\netc\ngames\ninclude`)
```